pkg/validate/method.go

```diff
@@ -2,6 +2,8 @@ package validate
 
 import "os"
 
+// IsAgentBasedInstallation determines whether we are using the 'agent'
+// subcommand to install.
```

Comment:
This will conflict with [my PR](https://github.com/openshift/installer/pull/6796) that deletes this code altogether, but if that's the price of stopping the bleeding I am ok with it :)

Reply:
I can also rebase this if you prefer that your PR gets merged first.